vagrant/provisioning/arkcase-dev-facts.yml

```diff
@@ -760,6 +760,8 @@ file_management_s3_buckets:
   - config_name: arkcaseBucket
     bucket_name: arkcase-portal
 
+prc_dockets_s3_bucket_name: prc-rca
```

Comment:
What is the right bucket name for PRC dev?  I'll have to update the AWX Tower configuraton.

Reply:
arkcase-fedramp-sandbox-file-management